pkg/provider/aws/client.go

```diff
@@ -94,6 +101,9 @@ type Client interface {
 	DescribeRouteTables(*ec2.DescribeRouteTablesInput) (*ec2.DescribeRouteTablesOutput, error)
 	DescribeSubnets(*ec2.DescribeSubnetsInput) (*ec2.DescribeSubnetsOutput, error)
 	DescribeVpcs(*ec2.DescribeVpcsInput) (*ec2.DescribeVpcsOutput, error)
+	DescribeVpcEndpoints(*ec2.DescribeVpcEndpointsInput) (*ec2.DescribeVpcEndpointsOutput, error)
```

Comment:
Just a note that there's an effort that's almost completed to port the AWS stuff to the v2 SDK: https://github.com/openshift/osdctl/pull/411

It might be worth updating this PR to that before merging.